app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> int:
+    player_reting = []
```

Comment:
The variable `player_reting` is declared but not used in the function. Consider removing it to clean up the code.